### DISCUSSION TITLE
High availability

### DISCUSSION THREAD

**Discussion by johnson-cloud** [dcm4che/dcm4chee-arc-light#4188] [General]
Dear all,

Any configuration or features available for setting up dcm4chee arc ui for high availability/failover?

Kindly update me or accept this as feature request please.

Highly appreciated for any assistance.

**Comment by johnson-cloud** [comment]
No update from long time?

**Comment by vrindanayak** [comment]
Yes, services for HA / failover are part of professional version of the archive
